diff --git U8g2_for_Adafruit_GFX/src/u8g2_fonts.h b/D:/Documents/Arduino/libraries/U8g2_for_Adafruit_GFX/src/u8g2_fonts.h
index 0be6aaa..17f0a7a 100644
--- U8g2_for_Adafruit_GFX/src/u8g2_fonts.h
+++ b/D:/Documents/Arduino/libraries/U8g2_for_Adafruit_GFX/src/u8g2_fonts.h
@@ -58,12 +58,6 @@
 #  define U8X8_PROGMEM PROGMEM
 #endif
 
-#if defined(__GNUC__) && defined(ESP8266)
-#  define U8X8_FONT_SECTION(name) U8X8_SECTION(".irom0.text." name)
-#  define u8x8_pgm_read(adr) pgm_read_byte_near(adr)
-#  define U8X8_PROGMEM PROGMEM
-#endif
-
 #ifndef U8X8_FONT_SECTION
 #  define U8X8_FONT_SECTION(name) 
 #endif
